05/styles/mobile.css

```diff
@@ -0,0 +1,74 @@
+
+/* Nav section */
+.nav {
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    position: relative;
+}
+/* Logo przeskakuje nie wiem dlaczego */
```

Comment:
Nie zauważyłem, aby przeskakiwało do gór jak napisałaś w komentarzu tylko do środka.
Jeśli o to chodzi to wystarczy zrezygnować z `align-items`.
Pamiętaj, że `flex-direction: column` powoduje odwrócenie całej osi o 90 stopni co powoduje, że teraz wspomniany `align-items` zarządza osią poziomą